edly_selfhosted/edly_selfhosted/models.py

```diff
@@ -3,3 +3,4 @@
 
 class OnboardingSurvey(models.Model):
     form_filled = models.BooleanField(default=False)
+    load_form = models.BooleanField(default=True)
```

Comment:
You need to add a comment to explain how this model works and what the different fields mean. Consider renaming the fields to make them more explicit.